WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/RestoreFragment.kt

```diff
@@ -89,7 +89,7 @@ class RestoreFragment : Fragment(R.layout.jetpack_backup_restore_fragment) {
                 site to activityId
             }
             else -> {
-                AppLog.e(T.JETPACK_RESTORE, "Error initializing ${this.javaClass.simpleName}")
+                AppLog.e(T.JETPACK_REWIND, "Error initializing ${this.javaClass.simpleName}")
```

Comment:
Question (❓): Why did this change to `REWIND`? Since rewind is used in general with backup download as well, as a term that is (like `rewindId`), I think it is better if we use `RESTORE` as a naming convention on restore specific cases. I try to do that everywhere, when refactoring field, parameters, functions, etc. Thus, I asking this question to understand if I am missing any information in regards to that.

Reply:
> Why did this change to REWIND?